index.html

```diff
@@ -900,91 +900,112 @@ <h4>Text Language Source</h4>
+  daptm:onScreen
+    : "ON"     # default
+    | "OFF"
+    | "ON_OFF"
+    | "OFF_ON"
+                    </pre>
+                  </div>
+                </li>
+              </ul>
+            </p>
+          </section>
+          <section>
+            <h5>Script Event Description</h5>
+            <p>The <dfn>Script Event Description</dfn> property is an annotation providing a human-readable description of a <a>Script Event</a>.</p>
+            <p>The <a>Script Event Description</a> property is represented in a <a>DAPT Document</a> by a <code>&lt;ttm:desc&gt;</code> element at the <code>&lt;div&gt;</code> level.</p>
+            <p class="note">The <a>Script Event Description</a> does not need to be unique, i.e. it does not need to have a different value for each <a>Script Event</a>. 
+              For example a particular value could be re-used to identify in a human-readable way one or more <a>Script Events</a> that are intended to be processed together,
+              e.g. in a batch recording.</p>
           <pre class="example"
-            data-include="examples/event-desc-multiple-descType.xml"
+            data-include="examples/event-desc.xml"
             data-include-format="text">
           </pre>
-        </section>
-        <section>
-          <h4>Script Event Type</h4>
-          <p>The <dfn>Script Event Type</dfn> property provides one or more space-separated keywords representing the type of the <a>Script Event</a>,
-            i.e. spoken text, or on-screen text,
-            and in the latter case, the type of on-screen text (title, credit, location, ...).
-            The possible keywords are indicated in the registry at XXXX.</p>
-          <p class="ednote">Registry to be defined.</p>
-          <p>The <a>Script Event Type</a> is represented in a <a>DAPT Document</a> by the following attribute:
+            <p>The <code>&lt;ttm:desc&gt;</code> element MAY have a <code>daptm:descType</code> attribute specified to indicate the type of description. The <code>daptm:descType</code> attribute is defined below. Its possible values are as indicated in the registry at YYY.</p>
+            <p class="ednote">Registry to be defined.</p>
             <div class="exampleInner">
               <pre class="language-abnf">
-daptm:eventType : string
+  daptm:descType : string
               </pre>
             </div>
-            <pre class="example">
-...
-&lt;div xml:id=&quot;event_1&quot;
-     begin=&quot;9663f&quot; end=&quot;9682f&quot; 
-     ttm:agent=&quot;character_4&quot;&gt;
-     daptm:eventType=&quot;dialogue&quot;
-...
-&lt;/div&gt;
-...
+            <pre class="example"
+              data-include="examples/event-desc-descType.xml"
+              data-include-format="text">
             </pre>
-          </p>
+            <p>Multiple <code>&lt;ttm:desc&gt;</code> elements MAY be present with different values of <code>daptm:descType</code>, as in the following example.</p>
```

Comment:
Thought in passing (maybe a separate issue): do we want to constrain that no more than one `ttm:desc` element with the same parent element has the same `daptm:descType` attribute value?